src/routes.js

```diff
@@ -27,6 +27,7 @@ export default function (props = {}) {
         <Route path='auth/login' component={LoginPage} />
         <Route path='auth/logout' component={Logout} />
         <Route path='petitions' component={Petitions} />
+        <Route path='petitions/page/:page' component={Petitions} />
```

Comment:
Just found out you can make the page param optional: `<Route path='petitions(/page/:page)' component={Petitions} />` and have only one route definition.

Will be helpfull when adding more params like `city` and `state`.